src/components/views/causes/filter/CausesFiltersButton.tsx

```
@@ -0,0 +1,54 @@
+	const delayedIsFilterOpen = useDelay(isFilterOpen, 280);
+
+	const filterMenuRef = useRef<HTMLDivElement>(null);
+
+	useOnClickOutside(
+		() => setIsFilterOpen(false),
+		isFilterOpen,
+		filterMenuRef,
+	);
+
+	const handleFilterClose = (e?: MouseEvent<HTMLElement>) => {
+		e?.stopPropagation && e?.stopPropagation();
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)